tests/test_datetime/conftest.py

```diff
@@ -0,0 +1,95 @@
+import pandas as pd
+import pytest
+
+
+@pytest.fixture(scope="module")
+def df_datetime_transformed(df_datetime):
+    today = pd.Timestamp.today()
+    data = {
+        "datetime_range_month": [2, 2, 2, 2],
+        "datetime_range_quarter": [1, 1, 1, 1],
+        "datetime_range_semester": [1, 1, 1, 1],
+        "datetime_range_year": [2020, 2020, 2020, 2020],
+        "datetime_range_wotm": [4, 4, 4, 4],
+        "datetime_range_woty": [9, 9, 9, 9],
+        "datetime_range_dotw": [0, 1, 2, 3],
+        "datetime_range_dotm": [24, 25, 26, 27],
+        "datetime_range_doty": [55, 56, 57, 58],
+        "datetime_range_weekend": [False, False, False, False],
```

Comment:
I am noticing that the default return for some of these datetime features is True and False. 

However, the idea of feature-engine's transformers, whenever possible, is to return numerical features that are ready ti use with sklearn estimators.

Could we add a workaround to transform these booleans into [0,1] binary vectors?